stripe/api_resources/abstract/verify_mixin.py

```diff
@@ -0,0 +1,10 @@
+from stripe import util
+
+
+class VerifyMixin(object):
+
+    def verify(self, idempotency_key=None, **params):
+        url = self.instance_url() + '/verify'
+        headers = util.populate_headers(idempotency_key)
+        self.refresh_from(self.request('post', url, params, headers))
+        return self
```

Comment:
IMO, this might be a little more abstraction than is necessary in that it saves some LOCs, but also makes the code more indirect and harder to follow. In practice the implementation ~never changes, so having it updatable in one place doesn't buy you all that much.

That said, given that it's already here, I think it'd be okay to leave it.